pkg/resolver/mocks/mock-rel-1.json

```diff
@@ -0,0 +1,18 @@
+{"addRecords":[
```

Comment:
This can be named just "records" (or "rows" if you want to match the postgres language).